package.json

```diff
@@ -17,5 +17,9 @@
     "tslib": "^2.6.0",
     "typescript": "^5.0.2",
     "vite": "^4.4.5"
+  },
+  "dependencies": {
+    "dotenv": "^16.3.1",
+    "exifr": "^7.1.3"
```

Comment:
Please commit dependencies as separate commit. Btw, `exifr` should have been part of previos PR isn't it ? :)
```
build(#12): add dotenv, exifr dependencies
```